src/test/java/de/tum/in/www1/artemis/service/JenkinsInternalUrlServiceTest.java

```diff
@@ -58,6 +58,9 @@ public void testGetVcsUrlOnInternalVcsUrl() {
         var vcsRepositoryUrl = mock(VcsRepositoryUrl.class);
         doReturn(null).when(vcsRepositoryUrl).getURL();
         assertThat(jenkinsInternalUrlService.toInternalVcsUrl(vcsRepositoryUrl)).isEqualTo(vcsRepositoryUrl);
+
+        String nullUrl = null;
```

Comment:
why this variable?

Reply:
The compiler complains when I try to write `null` directly.

Reply:
probably due to overloading. If you use null directly, the compiler can't tell which method to invoke, because there is no type given

Reply:
Yes exactly that

Reply:
As it's a test, I think that's not that critical so maybe just mark as resolved :)